scheduler/test/cook/test/config.clj

```diff
@@ -97,3 +97,33 @@
                                [:config :kubernetes :controller-lock-num-shards]
                                256)]
       (is (thrown? ExceptionInfo (config-settings bad-config))))))
+
+(deftest test-guard-invalid-gpu-models
+  (testing "test"
+    (is (nil? (config/guard-invalid-gpu-models))))
+  (testing "valid default model"
+    (with-redefs [config/valid-gpu-models (constantly [{:pool-regex    "test-pool"
```

Comment:
I think this is Cursive doing default formatting that we don't use:

```suggestion
    (with-redefs [config/valid-gpu-models (constantly [{:pool-regex "test-pool"
```

Reply:
done and changed the formatting in rest of the test as well